client/src/components/event-types/all-event-types/ActionBlock.vue

```diff
@@ -8,7 +8,9 @@
                 <VCol cols="10">
                     <span class="user-name">{{ user.name }}</span>
                     <br />
-                    <a href="">shedulia.xyz/nickname</a>
+                    <RouterLink :to="{ path: user.nickname }">
+                        shedulia.xyz/{{ user.nickname }}
```

Comment:
Add to language file `shedulia.xyz`

Reply:
How it will be translated?

Reply:
No, they not translate. I said wrong, you just need to output it as a constant.  I suggest adding to .env